### PR TITLE
New remove resource groups automation pack

### PR DESCRIPTION
**Pull request by dtzar**

### CODE REVIEW

File: 101-remove-resourcegroups/DeployThroughARM.ps1 [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+#Set the parameter values for the template
+$Params = @{
+    "accountName" = "MyAccount2" ;
+    "regionId" = "Japan East";
+	"userName" = "Your UserName"; 
+	"password" = "Your password";
+}
+
+$TemplateURI = "https://raw.githubusercontent.com/azureautomation/automation-packs/master/101-remove-resourcegroups/azuredeploy.json"
```
Comment (elcooper) [inline 101-remove-resourcegroups/DeployThroughARM.ps1:19]: You may want to use a permalink here.  Not really a big deal, but I've noticed sometimes the URLS change which will would make this fail.  See https://help.github.com/articles/getting-permanent-links-to-files/ for getting GitHub permalink.

File: 101-remove-resourcegroups/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Remove ResourceGroups runbook and Automation credential
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazureautomation%2Fautomation-packs%2Fmaster%2F101-remove-resourcegroups%2Fazuredeploy.json)
```
Comment (elcooper) [inline 101-remove-resourcegroups/README.md:2]: Same comment as above...may want to use a permalink

**Comment by elcooper** [comment]
Thanks for putting this together!  I added a comment to this review in case you want to make the update, but this looks good.

**Comment by dtzar** [comment]
I'd like to keep the links as-is so if we update the underlying script / template it will be automatically reflected vs tying to a specific commit.

**Comment by dtzar** [comment]
Wasn't sure what to call it 101/102 etc with the naming here, but if it works for you - then all good :)

**Comment by dtzar** [comment]
One note - you might want to update the API version for some of your other templates using the 2015-01-01-preview versus the newer 2015-10-31 version.

**Comment by elcooper** [comment]
Thanks David!  I’d been meaning to update the schema version, but just hadn’t gotten around to it yet.  ☺

**Pull request merged**